packages/maskbook/src/plugins/ITO/UI/StyledLinearProgress.tsx

```diff
@@ -1,13 +1,26 @@
-import { withStyles, LinearProgress } from '@material-ui/core'
+import { LinearProgress, LinearProgressProps } from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles'
 
-export const StyledLinearProgress = withStyles({
-    root: {
-        height: 8,
-        borderRadius: 5,
-        background: 'rgba(255, 255, 255, 0.3)',
-    },
-    bar: {
-        borderRadius: 5,
-        backgroundColor: '#fff',
-    },
-})(LinearProgress)
+interface StyledLinearProgress extends LinearProgressProps {
+    barcolor?: string
+    backgroundcolor?: string
+}
+
+const useStyles = makeStyles(() => {
+    return {
+        root: {
+            height: 8,
+            borderRadius: 5,
+            backgroundColor: (props: StyledLinearProgress) => props.backgroundcolor ?? 'rgba(255, 255, 255, 0.3)',
+        },
+        bar: {
+            borderRadius: 5,
+            backgroundColor: (props: StyledLinearProgress) => props.barcolor ?? '#fff',
```

Comment:
does it fit dark theme?

Reply:
It fits well. 

<img src="https://user-images.githubusercontent.com/63733714/108017352-a0f3ed80-704f-11eb-9958-756cd29128d8.png" width= 600 />

<img src="https://user-images.githubusercontent.com/63733714/108017378-b49f5400-704f-11eb-95e2-d2a294559c90.png" width= 600 />